ctms/crud.py

```diff
@@ -79,25 +79,55 @@ def _contact_base_query(db):
     )
 
 
+def get_bulk_query(start_time, end_time, after_email_uuid, mofo_relevant):
```

Comment:
Now that there are three paths, I think there is a higher risk of common changes not being reflected across all three options. Can you combine the common path, something like this:

```python
filters = [
  Email.update_timestamp >= start_time,
  Email.update_timestamp < end_time,
  Email.email_id != after_email_uuid
]
if mofo_relevant is False:
  filters.append(_or(Email.mofo == None, Email.mofo.has(mofo_relevant=mofo_relevant))
if mofo_relevant is True:
  filters.append(Email.mofo.has(mofo_relevant=mofo_relevant))
return filters
```